order-service/src/entities/order.entity.ts

```diff
@@ -0,0 +1,83 @@
+import {
+    Column,
+    Entity,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+    ManyToOne,
+} from "typeorm";
+import { Length } from "class-validator";
+import Messages from "../utils/messages";
+
+/**
+ * @namespace Entities
+ * @class Order
+ */
+@Entity()
+export class Order {
+    @PrimaryGeneratedColumn()
+    public id: number;
+
+    @Column()
+    public userId: number;
+
+    @Column()
+    public placeId: number;
+
+    @Column()
+    public checkoutSlotId: number;
+
+    @Column({ length: 64 })
+    @Length(3, 255, {
+        message: Messages.validation.order_status_length,
+    })
+    public status: string;
```

Comment:
I think the status should be a number. Create a constant to hold the values we want for the status:

```ts
const STATUS_VALUE = {
    0: "Novo",
    1: "Em preparo",
    2: "Aguardando retirada"
}
``` 

When declaring the `status` property, you can validate its value like this:
```ts
@Column()
@IsIn(Object.keys(STATUS_VALUE), {
    message: Message.validation.order_status_dont_exist,
})
public status: number;
```

Reply:
Perfect

Reply:
Done